core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -289,4 +289,9 @@ default boolean shutdownBlockingTaskExecutorOnStop() {
      * supports case sensitive HTTP/1 headers.
      */
     Http1HeaderNaming http1HeaderNaming();
+
+    /**
+     * Returns the {@link DependencyInjector} that injects dependencies in annotated services.
```

Comment:
Should the `DependencyInjector` be used in a gRPC service?

Reply:
That's a good point. 😉 We might support adding annotations to a `Thrift` service later so let me change the sentence to `injects dependencies in annotations`